File: Bee-Crowd/Exercises_C++/1101.cpp
#include <iostream>
#include <cmath> 
using namespace std;

int main() {
    int x, y, maior, menor;
    cin >> x >> y;

    while (x > 0 && y > 0) {
        maior = (x + y + abs(x - y)) / 2;
        menor = (x + y - abs(x - y)) / 2;

        
        int tamanho = maior - menor + 1;
        int vetor[tamanho];
        int soma = 0;

        for (int c = 0; c < tamanho; c++) {
            vetor[c] = menor + c;
            soma += vetor[c];
        }


        for (int c = 0; c < tamanho; c++) {
            cout << vetor[c] << " ";
        }

        cout << "Sum=" << soma << endl;

        cin >> x >> y;
    }

    return 0;
}

File: Bee-Crowd/Exercises_Python/1101.py
a, b = map(int, input().split())

while (a > 0) and (b > 0):
    menor = min(a,b)
    maior = max(a,b)
    valores = list(range(menor, maior + 1))
    soma = sum(valores)
    print(' '.join(map(str, valores)),' ','Sum=',soma, sep='')
    a, b = map(int, input().split())